src/object-store/src/cache_policy.rs

```diff
@@ -19,12 +19,18 @@ use std::sync::Arc;
 use async_trait::async_trait;
 use bytes::Bytes;
 use lru::LruCache;
+use metrics::increment_counter;
 use opendal::ops::{OpDelete, OpList, OpRead, OpScan, OpWrite};
 use opendal::raw::oio::{Read, Reader, Write};
 use opendal::raw::{Accessor, Layer, LayeredAccessor, RpDelete, RpList, RpRead, RpScan, RpWrite};
 use opendal::{ErrorKind, Result};
 use tokio::sync::Mutex;
 
+const METRIC_LRU_CACHE_NAME_LABEL: &str = "lru_cache.name";
```

Comment:
Prefer to move these metric names to a separate sub-mod such as `metrics`.

You can find the example at https://github.com/GreptimeTeam/greptimedb/blob/develop/src/mito/src/metrics.rs

And I think we can add a `object_store.`prefix to all these metric names.

Reply:
Done.